package.json

```diff
@@ -33,11 +33,13 @@
     "ember-disable-prototype-extensions": "^1.1.0",
     "ember-disable-proxy-controllers": "^1.0.1",
     "ember-export-application-global": "^1.0.4",
+    "ember-get-config": "0.0.2",
```

Comment:
This needs to be in `dependencies` instead of `devDependencies` so consumers don't get errors that it is missing.